tests/monitoring/vm_monitoring.go

```diff
@@ -48,7 +48,7 @@ import (
 	"kubevirt.io/kubevirt/tests/flags"
 	"kubevirt.io/kubevirt/tests/framework/checks"
 	"kubevirt.io/kubevirt/tests/framework/kubevirt"
-	"kubevirt.io/kubevirt/tests/framework/matcher"
+	. "kubevirt.io/kubevirt/tests/framework/matcher"
```

Comment:
I think we should avoid dot import if we don't have to use it. here it's not really needed, TMHO. In general, dot import is a bad practice.

Reply:
@Barakmor1 PTAL

Reply:
I noticed that we're always using the dot import for this package in other files, so I believe it would be better for consistency. 
@xpivarc @EdDev WDYT?

Reply:
Done.